features/serializable-errors.yml

```
@@ -0,0 +1,13 @@
+name: Serializable errors
+description: "`DOMException`, `Error`, `EvalError`, `RangeError`, `ReferenceError`, `SyntaxError`, `TypeError`, and `URIError` are serializable (structuring cloning is possible). This allows you to pass errors from a worker using `postMessage()`."
```

I thought we could work the `structuredClone` bit in a little more smoothly.

```suggestion
description: "The `DOMException`, `Error`, `EvalError`, `RangeError`, `ReferenceError`, `SyntaxError`, `TypeError`, and `URIError` objects are serializable. You can call `structuredClone()` on an error object or pass it to a worker using `postMessage()`."
```